DM-39097: Start a ci_summit repo

hsinfang opened the pull request.

hsinfang committed:
Add a raw file from from /repo/main the LATISS-test-data collection

hsinfang committed:
Add boilerplate

Mostly copied from templates/project_templates/stack_package/example_dataonly

czwa reviewed (approved):
I would suggest adding a more useful description to the `README.rst` file, even though when I went to look at one of my own packages (`testdata_latiss_cpp`) I found the same stub there as well.
Not having a top-level `SConstruct` file means that the documentation under `doc` will never be generated.
Neither of these are blockers to me.

hsinfang committed:
Add more description to README

hsinfang committed:
Add two more raw files

AT_O_20200315_000120--R00S00.fit is what test_imageExaminer.py
in summit_utils currently uses.

Add one more raw from 20210121 to help test nightReport.
